src/AboutUs/View/page.twig

```diff
@@ -3,7 +3,35 @@
 	<body class="top-wave">
 	{% endblock %}
 	<section class="container">
-		<h1>O nas</h1>
+		<h1 class="u-center-text u-margin--small">Historia naszego teamu "DevSquad</h1>
+			<div>
+				<p> It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using ‘Content here, content here’, making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for ‘lorem ipsum’ will uncover many web sites still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose (injected humour and the like).</p>
+				<p>There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected humour, or randomised words which don’t look even slightly believable. If you are going to use a passage of Lorem Ipsum, you need to be sure there isn’t anything embarrassing hidden in the middle of text. All the Lorem Ipsum generators on the Internet tend to repeat predefined chunks as necessary, making this the first true generator on the Internet. It uses a dictionary of over 200 Latin words, combined with a handful of model sentence structures, to generate Lorem Ipsum which looks reasonable. The generated Lorem Ipsum is therefore always free from repetition, injected humour, or non-characteristic words etc.</p>
+			</div>
+		<h2 class="u-center-text u-margin--medium">Poznaj nas bliżej</h2>
+			<div class="row clearfix">
+				<div class="col-sm-3 u-margin--small person-box">
+					<img class"person-box__img" src=" /img/person1.png"></img>
+					<div class="person-box__description">
+						<h4 class="u-center-text u-margin-small">Ryszard Okucki</h4>
+						<p>Miś</p>
+					</div>	
+				</div>
+				<div class="col-sm-3 u-margin--small person-box">
+					<img class"person-box__img" src=" /img/person2.png"></img>
+					<div class="person-box__description">
+						<h4 class="u-center-text u-margin-small">Aniołowa</h4>
+						<p>Alternatywy 4</p>
+					</div>	
+				</div>
+				<div class="col-sm-3 u-margin--small person-box">
+					<img class"person-box__img" src=" /img/person3.png"></img>
```

Comment:
There is no equal sign `class"person-box__img"`. Change it to `class="person-box__img"`

Reply:
Thank you!!! I haven't seen it before. What a relief :)